cluster/examples/kubernetes/ceph/object-bucket-claim-delete.yaml

```
@@ -11,3 +11,7 @@ spec:
   #bucketName: 
   generateBucketName: ceph-bkt
   storageClassName: rook-ceph-delete-bucket
+  #additionalConfig:
```

Is there any impact to having this uncommented by default? The individual settings would just be commented.
```suggestion
  additionalConfig:
```

---

Sure @jeffvance recommended this way in my previous patch set

---

It's fine to leave the stanza name un-commented and comment out the values.